scissors/scissors_game/scissors.h

```diff
@@ -0,0 +1,19 @@
+/*Enum of Game results*/
+typedef enum {
+	eDRAW = -1,
+	eLOST,
+	eWINN
+} decision_t;
+/*Enum of Unknown, Rock, Paper and Scissors choise*/
+typedef enum {
+	eU = -1,
+	eR,
+	eP,
+	eS
+} element_t;
+const int game_table[3][3]={
+       eDRAW, eLOST, eWINN,
+       eWINN, eDRAW, eLOST,
+       eLOST, eWINN, eDRAW
+};
```

Comment:
You shouldn't define variables in header files.
This will lead to error in case of it's inclusion in more than one source file (which is actually what headers are for).
The table should be moved into separate source file.
Together with function for comparison (and probably function for parsing as well).